beamer/tests/health/conftest.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+import pytest
+
+from beamer.health.check import Context
+from beamer.health.notification_state import NotificationState
+from beamer.tests.health.test_check import AGENT_ADDRESS
+
+config = {
+    "agent_address": "",
+    "deployment-dir": "../deployments/mainnet",
+    "notification_system": "rocketchat",
+    "rpcs": {
+        10: "https://example.rpc.com",
+        42161: "https://example.rpc.com",
+    },
+    "explorers": {
+        10: "https://optimistic.etherscan.io/tx/",
+        42161: "https://arbiscan.io/tx/",
+    },
+    "notification": {
+        "rocketchat": {
+            "url": "https://example.com",
+            "channel": "beamer",
+        },
+        "telegram": {
+            "token": "",
+            "chat_id": "",
+        },
+    },
+}
+
+
+@pytest.fixture(autouse=True)
+def get_config(monkeypatch):
+    """Set default config for script."""
+    monkeypatch.setattr("beamer.health.check.get_config", lambda: config)
+
+
+@pytest.fixture
+def ctx():
+    ctx = Context(agent_address=AGENT_ADDRESS)
+    ctx.notification_state = NotificationState(Path("./"))
```

Comment:
This will leave `breadcrumbs.json` wherever the current working directory happens to be, right?
Wouldn't it be better if we used a temporary directory that gets cleaned up after the tests are done?
Please take a look at https://docs.pytest.org/en/6.2.x/tmpdir.html#temporary-directories-and-files.

Reply:
Well, no. Because I have to explicitly call notification_state.persist for the breadcrumbs file to be written do the filesystem and currently the tests in the PR don't test this part of the code.

Reply:
> Well, no. Because I have to explicitly call notification_state.persist for the breadcrumbs file to be written do the filesystem and currently the tests in the PR don't test this part of the code.

Ah, I see. I'm ok with not having tests for that part just yet,
but I would still prefer to use a temporary path here, like
```python
@pytest.fixture
def ctx(tmp_path):
    ctx = Context(agent_address=AGENT_ADDRESS)
    ctx.notification_state = NotificationState(tmp_path)
```